[2024-11-29 11:17:32,152] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: kpi_calculation_pipeline.store_kpi 2024-11-22T17:05:00+00:00 [queued]>
[2024-11-29 11:17:32,213] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: kpi_calculation_pipeline.store_kpi 2024-11-22T17:05:00+00:00 [queued]>
[2024-11-29 11:17:32,213] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-11-29 11:17:32,214] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2024-11-29 11:17:32,215] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-11-29 11:17:32,303] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): store_kpi> on 2024-11-22T17:05:00+00:00
[2024-11-29 11:17:32,308] {standard_task_runner.py:52} INFO - Started process 240 to run task
[2024-11-29 11:17:32,312] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'kpi_calculation_pipeline', 'store_kpi', '2024-11-22T17:05:00+00:00', '--job-id', '379', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/data_injestion_dag/bazefield_kpi_calculation.py', '--cfg-path', '/tmp/tmpnis5ck8b', '--error-file', '/tmp/tmpxawa93i0']
[2024-11-29 11:17:32,314] {standard_task_runner.py:77} INFO - Job 379: Subtask store_kpi
[2024-11-29 11:17:32,464] {logging_mixin.py:104} INFO - Running <TaskInstance: kpi_calculation_pipeline.store_kpi 2024-11-22T17:05:00+00:00 [running]> on host 3f28a2ef9091
[2024-11-29 11:17:32,692] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=kpi_calculation_pipeline
AIRFLOW_CTX_TASK_ID=store_kpi
AIRFLOW_CTX_EXECUTION_DATE=2024-11-22T17:05:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-11-22T17:05:00+00:00
[2024-11-29 11:17:32,740] {base.py:69} INFO - Using connection to: id: postgres_default. Host: 10.197.48.9, Port: 5432, Schema: kpi_data, Login: rakshit, Password: ***, extra: {}
[2024-11-29 11:17:32,910] {logging_mixin.py:104} INFO - successfully connected
[2024-11-29 11:17:32,988] {python.py:151} INFO - Done. Returned value was: None
[2024-11-29 11:17:33,010] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=kpi_calculation_pipeline, task_id=store_kpi, execution_date=20241122T170500, start_date=20241129T111732, end_date=20241129T111733
[2024-11-29 11:17:33,122] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-29 11:17:33,148] {local_task_job.py:151} INFO - Task exited with return code 0
